_includes/specials/bech32/22-priority.md

```diff
@@ -0,0 +1,35 @@
+We've frequently mentioned the fee savings available to people spending
+segwit inputs, but we've never before mentioned that you don't need to
+take advantage of the savings.  If you want, you can pay the same fee
+you would've paid without segwit in order to possibly have your
+transaction confirm quicker (all other things being equal).  For some
+users, such as traders attempting arbitrage across exchanges, saving
+money may not be as important as getting transactions confirmed quicker
+for the same amount of money.
```

Comment:
suggestion: s/getting transactions confirmed quicker for the same amount of money/fast transaction confirmations/